src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 48efe97ab9de41547a64fd2b63d9c457
+guid: 2a03537a421f0144489f130406659dc5
```

Comment:
The mysterious meta file changes continue...

Reply:
i'm going to revert this.